src/executor.rs

```diff
@@ -726,10 +727,10 @@ impl<T> Task<T> {
     /// # Examples
     ///
     /// ```
-    /// use async_executor::{Executor, Task};
+    /// use scipio::{LocalExecutor, Timer};
     /// use futures_lite::future;
     ///
-    /// let ex = Executor::new();
+    /// let ex = LocalExecutor::new(None).expect("failed to create local executor");
```

Comment:
my understanding is that `yield_now` blocks the thread, which is why we have our own version, called `later()`.
At the very least we should use `later()` in the examples.

I would like to see this one fixed, please add a commit on top addressing this.